src/screens/MainScreen/components/AddCartForm/components/ColorPicker/index.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from "react";
+import {
+  Container,
+  ColorCircleContainer,
+  ColorCircle,
+  ColorText,
+  Arrow
+} from "./styles";
+import { CustomPicker, BlockPicker } from "react-color";
+import OverlayTrigger from "react-bootstrap/OverlayTrigger";
+import Popover from "react-bootstrap/Popover";
+
+const ColorPicker = ({ hex, onChange }) => {
+  const [color, setColor] = useState(hex);
+
+  const handleChange = color => {
+    onChange(color.hex);
+    setColor(color.hex);
+  };
+
+  const picker = (
+    <Popover>
+      <Popover.Title>Pick a color!</Popover.Title>
+      <BlockPicker triangle="hide" color={color} onChange={handleChange} />
```

Comment:
do we want to let people choose their own hex values for colors? (i couldn't get this to work)

Reply:
This form works the way you'd expect because the `BlockPicker` doesn't let you choose your own. Ideally, though, we'd wanna implement it as another picker, but I left it as a new PR for now. See #14